src/plugins/intel_gpu/src/graph/graph_optimizer/prepare_primitive_fusing.cpp

```diff
@@ -991,8 +991,24 @@ void prepare_primitive_fusing::fuse_simple_primitives(program &p) {
             auto p1_dt = parents[fused_idx].first->get_output_layout().data_type;
             auto p2_dt = parents[peer_idx].first->get_output_layout().data_type;
 
+            bool is_node_in_fusing_history = false;
+            for (auto fused_prim : parents[fused_idx].first->get_fused_primitives()) {
+                if (is_node_in_fusing_history)
+                    break;
+                auto iter = fusing_history.find(node.id());
+                if (iter != fusing_history.end()) {
+                    for (auto id : iter->second) {
+                        if (id.first == fused_prim.desc->id) {
+                            is_node_in_fusing_history = true;
+                            break;
+                        }
+                    }
+                }
+            }
```

Comment:
And I think we need one more condition that the dep in the fusing history is not the last node in the fused_primitives in the fused_node. 

Also could you please add an description of the target issue here? 
(with a diagram)

Reply:
Added additional condition and description.